plugins/plugin-hosted/codenvy-hosted-ext-hosted/pom.xml

```diff
@@ -25,8 +26,16 @@
     <artifactId>codenvy-hosted-ext-hosted</artifactId>
     <packaging>jar</packaging>
     <name>Codenvy Plugin :: Hosted Environment Tuning</name>
+    <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+    </properties>
     <dependencies>
         <dependency>
+            <groupId>com.codenvy.hosted-infrastructure</groupId>
+            <artifactId>codenvy-hosted-workspace-activity</artifactId>
```

Comment:
I think it is not worth it to create a separate shared module for 2 constans.
How about I just place Constants class in separate package?